bitnami/kafka/3.2/debian-11/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -772,8 +772,8 @@ kafka_zookeeper_configure_tls() {
     elif [[ "$KAFKA_ZOOKEEPER_TLS_TYPE" = "PEM" ]] && [[ -f "$KAFKA_CERTS_DIR"/zookeeper.keystore.pem ]] && [[ -f "$KAFKA_CERTS_DIR"/zookeeper.keystore.key ]]; then
         # Concatenating private key into public certificate file
         # This is needed to load keystore from location using PEM
-        cat "$KAFKA_CERTS_DIR"/zookeeper.keystore.key >>"$KAFKA_CERTS_DIR"/zookeeper.keystore.pem
-        keystore_location="${KAFKA_CERTS_DIR}/zookeeper.keystore.pem"
+        cat "$KAFKA_CERTS_DIR"/zookeeper.keystore.pem "$KAFKA_CERTS_DIR"/zookeeper.keystore.key > "$KAFKA_CERTS_DIR"/zookeeper-keypair.pem
+        keystore_location="$KAFKA_CERTS_DIR"/zookeeper-keypair.pem
```

Comment:
What do you think about this minor suggestion?

```suggestion
        keystore_location="${KAFKA_CERTS_DIR}/zookeeper.keypair.pem"
        cat "${KAFKA_CERTS_DIR}/zookeeper.keystore.pem" "${KAFKA_CERTS_DIR}/zookeeper.keystore.key" > "$keystore_location"
```

Could you apply it to the rest of the files?